installer/deb/scripts/postinst

```diff
@@ -57,8 +57,26 @@ set_local_dns() {
     echo "DNS configuration complete."
 }
 
+wait_for_network() {
+    counter=0
+    timeout=20  # Timeout in seconds
+
+    while ! ifconfig | grep "192.168." > /dev/null && ((counter < timeout)); do
+        echo "Waiting for network manager to be up and running"
+        sleep 1
+        ((counter+=1))
+    done
+
+    if ((counter >= timeout)); then
+        echo "Timeout reached. Network manager is not available."
```

Comment:
You should use `fatal` to print the error and stop the installation since there is a network issue

```suggestion
        fatal "Network manager is not available: Timeout reached. "
```